streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/SavePointServiceImpl.java

```diff
@@ -333,6 +333,16 @@ private void handleSavepointResponseFuture(
               if (savepointResponse != null && savepointResponse.savePointDir() != null) {
                 applicationLog.setSuccess(true);
                 String savePointDir = savepointResponse.savePointDir();
+
+                SavePoint savePoint = new SavePoint();
```

Comment:
Maybe you can try to optimize this part of the code

Reply:
What type of optimization do I need? I refer to the code from here: 
https://github.com/apache/incubator-streampark/blob/99d536474f79d4fce61ea6290e87246edeba4804/streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationActionServiceImpl.java#L364

Reply:
It seems that it can no longer be optimized, thank you